### ISSUE TITLE
Cannot login using docker-cli

### ISSUE THREAD

**Issue by shinebayar-g** [goharbor/harbor#5980]
**Expected behavior and actual behavior:**
I have just setup goharbor and it's awesome!
I've followed installation instructions. I'm putting it behind nginx proxy of host machine, not container's direct nginx. So I had to follow this #3114 post and changed nginx settings as well.
Now I can login to my web page without problem. But I cannot login same credentials using docker-cli?
```
It says Error response from daemon: Get https://goharbor.mydomain.com/v2/: unauthorized: authentication required (domain is censored)
```
I've changed default admin password, even resetted the password with lost password function.

**Steps to reproduce the problem:**
Install fresh new goharbor, put it behind nginx proxy of host machine, website works. But cannot login using docker-cli.

**Versions:**
- harbor version: [1.6.0]
- docker engine version: [18.06.1-ce]
- docker-compose version: [1.22.0, build f46880fe]

**Edit:** I even added into my insecure registry list by adding

```
{
    "insecure-registries" : [ "goharbor.mydomain.com" ]
}
```
into the file `/etc/docker/daemon.json`

and running
```
sudo systemctl daemon-reload
sudo systemctl restart docker
``` 
after wards. But still same.

**Comment by clouderati** [comment]
Hi @shinebayar-g – thanks for giving Harbor a spin. :+1:

Out of curiosity, does it work if you remove the `nginxg` proxy in front of the Harbor stack?

**Comment by shinebayar-g** [comment]
Okay, I successfully resolved issue using #3114 issue's https://github.com/goharbor/harbor/issues/3114#issuecomment-414912832 and https://github.com/goharbor/harbor/issues/3114#issuecomment-424992795 comments helped a lot!

However when I edit `common/config/registry/config.yml` file and change realm http -> https
Then if I run install.sh script it overrides the changes, so I just restarted the all docker processes.
